lib/solidus_page_objects/components/cart.rb

```diff
@@ -0,0 +1,34 @@
+module SolidusPageObjects
+  module Components
+    module Cart
+      def self.included(base)
+        base.sections :items, Sections::CartItem, '.line-item'
+
+        base.element :cart_total, ".cart-total"
+
+        base.element :empty_cart_button, "#empty-cart input[type='submit']"
+        base.element :continue_shopping_link, "#empty-cart a.continue"
+
+        base.element :coupon_code_input, "#order_coupon_code"
+        base.element :update_button, "#update-button"
+        base.element :checkout_link, "#checkout-link"
+      end
+
+      def apply_coupon_code(code)
+        coupon_code_input.set(code)
+        update_button.click
+        Pages::Cart.new
+      end
+
+      def update_quantity_for_item(index, quantity:)
```

Comment:
@mtylty so you can use the method like this `page.cart.update_quantity_for_item(0, quantity: 3)`, if we use position arguments you'll use the method like this `page.cart.update_quantity_for_item(0, 3)` but the first way seems more clear from my point of view